handbook/communication/content_guidelines/terminology_guidelines.md

```diff
@@ -92,7 +92,7 @@ Only use _we_ and _our_ (as in “our GitHub integration”) in informal documen
 |PostgreSQL   |Postgres, postgres, PgSQL, Postgresql, PostGres,   |Proper name   |
 |**R**   |   |   |
 |regex   |"regexp" or "regular expressions"   |This is to avoid confusion with RegExp in Javascript, and to avoid conflation with "exp" in Warcraft and other MMORPGs. This is relevant as we onboard younger users whose gaming experience is significant. If they're used to "exp" being experience, they will have to pause for a moment to context-swtich, thus creating confusion, even temporarily, which isn't a good look.|
-|repository   |repo   |   |
+|repository   |repo   | We try to [avoid abbreviations, acryonyms, latinisms, and jargon when possible](/communication/content_guidelines/style_and_mechanics#abbreviations-acronyms-latinisms-jargon). We habitually shorten words for concepts we work with every day until they become obvious to us, like repo, org, j2d, find-refs, PR, revs... but they aren't obvious to outsiders, which can be confusing and result in a steeper learning curve for new users. In this spirit, we prefer "repository" and "respositories". |
```

Comment:
We can probably shorten this since it's not a latinism etc just an abbreviation.

```suggestion
|repository   |repo   | We try to [avoid abbreviations when possible](/communication/content_guidelines/style_and_mechanics#abbreviations-acronyms-latinisms-jargon). We habitually shorten words for concepts we work with every day until they become obvious to us, like repo, org, j2d, find-refs, PR, revs... but they aren't obvious to outsiders, which can be confusing and result in a steeper learning curve for new users. In this spirit, we prefer "repository" and "respositories". |
```

Reply:
I also just noticed we actually say to use "org" over "organization" in the very same table. But we don't seem to follow it at all, so I don't know why it says that.

![image](https://user-images.githubusercontent.com/10532611/105344979-f8837280-5be3-11eb-9b22-88eb4b564530.png)

![image](https://user-images.githubusercontent.com/10532611/105345012-020cda80-5be4-11eb-9228-afcb428926ea.png)

Reply:
I wonder if they were inadvertently swapped?